src/linear_algebra/quotient.lean

```diff
@@ -206,6 +206,16 @@ def mkq : M →ₗ[R] M ⧸ p :=
 
 @[simp] theorem mkq_apply (x : M) : p.mkq x = quotient.mk x := rfl
 
+@[simp] lemma mk_restrict_scalars {S : Type*} [ring S] [has_scalar S R] [module S M]
+  [is_scalar_tower S R M] (P : submodule R M) :
+  (quotient.mk : M → M ⧸ P.restrict_scalars S) = (quotient.mk : M → M ⧸ P) :=
+rfl
```

Comment:
Should we instead have `M ⧸ P.restrict_scalars S = M ⧸ P` as the lemma statement?

Reply:
Hmm, rewriting in the type of data is a bit iffy. Maybe I should just use `quotient.restrict_scalars_equiv`.